discovery/discovery.go

```diff
@@ -10,23 +14,92 @@ import (
 	"github.com/golang/glog"
 )
 
-type offchainOrchestrator struct {
-	uri   *url.URL
+const GetOrchestratorsTimeoutLoop = 3 * time.Second
+
+type offchainOrchestrators struct {
+	uri   []*url.URL
 	bcast server.Broadcaster
 }
 
-func NewOffchainOrchestrator(node *core.LivepeerNode, address string) *offchainOrchestrator {
-	addr := "https://" + address
-	uri, err := url.ParseRequestURI(addr)
+func NewOffchainOrchestrator(node *core.LivepeerNode, addresses []string) *offchainOrchestrators {
```

Comment:
Agreed that the naming is a bit imprecise. Generally, things are getting to be a mouthful, such as with `NewOffchainOrchestratorFromOnchainList`.

Trying to think of more concise options. Perhaps `NewOrchestratorSelector` for this base case since the offchain part is a given anyway, and `NewOnchainOrchestratorSelector` for orchestrators registered on-chain? Is there a better name for the interface than `OrchestratorSelector` ?

Reply:
Perhaps `OrchestratorPool`, since it's a bit shorter? :)

Reply:
I love it 👍

Reply:
Agree that OrchestratorPool conveys meaning better than OrchestratorSelector! I'll change